src/itemEditableInfo/index.unit.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import ItemEditableIInfo from 'itemEditableInfo'
+
+it('Should render a basic ItemEditableInfo', () => {
+  const props = {
+    label: 'My label',
+    value: 'My value',
+    href: '/myhref',
+  }
+  const wrapper = mount(<ItemEditableIInfo {...props} />)
+  const anchor = wrapper.find('a')
```

Comment:
Avoid targeting tags

Reply:
humm I do not agree, I need to verify that we have an anchor with a href and a good value.

Reply:
I agree with @grosbouddha on this one but I think there's a need for a conversation on how we write test.

Reply:
I added an entry for this in the next chapter weekly minutes. I only wrote my name but everyone is free to speak.